src/SqlPersistence/Outbox/SqlPersistenceOutboxExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace NServiceBus
+{
+    using Configuration.AdvancedExtensibility;
+    using System;
+
+    /// <summary>
+    /// Contains extensions methods which allow to configure SQL persistence specific outbox configuration
+    /// </summary>
+    public static class SqlPersistenceOutboxExtensions
+    {
+        /// <summary>
+        /// Sets the time to keep the deduplication data to the specified time span.
+        /// </summary>
+        /// <param name="configuration">The configuration being extended.</param>
+        /// <param name="timeToKeepDeduplicationData">The time to keep the deduplication data.
+        /// The cleanup process removes entries older than the specified time to keep deduplication data. The time span cannot be negative or zero.</param>
+        /// <returns>The configuration</returns>
+        public static void SetTimeToKeepDeduplicationData(this EndpointConfiguration configuration, TimeSpan timeToKeepDeduplicationData)
```

Comment:
It is not obvious that we set this for the `outbox` feature. Can we make these methods extend `OutboxSettings` ?

Reply:
right, I intended to change that and forgot, fixing now!